### PR TITLE
Fix hit merging and add thresholds for backward HCal

### PR DESCRIPTION
**Pull request by lkosarz**
### Briefly, what does this PR introduce?

Implemented 100 ns integration time
Eth=0.25 MIP=0.1875 MeV for backward HCal
Fixed hit merging and plugged into clustering

### What kind of change does this PR introduce?
- [x] Bug fix (fixes no merging)
- [x] New feature (updated thresholds)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### CODE REVIEW

File: src/detectors/EHCAL/EHCAL.cc [hunk 1]
```diff
@@ -62,16 +63,16 @@ extern "C" {
           {
             .readout = "HcalEndcapNHits",
             .fields = {"layer", "slice"},
-            .refs = {1, 0},
+            .refs = {4, 0},
```
Comment (wdconinc) [inline src/detectors/EHCAL/EHCAL.cc:66]: Can you document the reason for `4` here?

Comment (lkosarz) [reply src/detectors/EHCAL/EHCAL.cc:66]: This is in order to place the hits in the middle at ~1 interaction length rather than in the front.

Comment (wdconinc) [reply src/detectors/EHCAL/EHCAL.cc:66]: I meant add a comment in the source code.

**Comment by wdconinc** [comment]
I have asked @kkauder to review this so he can keep the overview of thresholds and backgrounds in the context of the master spreadsheet. That's all we're waiting for.

**Review by kkauder — APPROVED** [review]

**Pull request merged**